src/ocm/components/newFeatureSupportLevels/newFeatureStateUtils.ts

```diff
@@ -91,8 +95,15 @@ const getCnvDisabledReason = (activeFeatureConfiguration: ActiveFeatureConfigura
   if (!activeFeatureConfiguration) {
     return undefined;
   }
-  if (activeFeatureConfiguration.underlyingCpuArchitecture === CpuArchitecture.ARM) {
-    return `${CNV_OPERATOR_LABEL} is not available when ARM CPU architecture is selected.`;
+  if (activeFeatureConfiguration.underlyingCpuArchitecture) {
```

Comment:
Wouldn't this make CNV disabled for ALL CPU Architectures?

Could this logic not be now implemented using the new FeatureSUpportLevels API? We didn't have the data earlier, but now it would be the best way to implement it and not have to update it for new CPU architectures.

Reply:
The data of the `underlyingCpuArchitecture` is being populated from the infraEnv OR the cluster, meaning it's always going to have some value.

```
  const activeFeatureConfiguration = React.useMemo<ActiveFeatureConfiguration>(
    () => ({
      underlyingCpuArchitecture: (infraEnv?.cpuArchitecture ||
        cluster?.cpuArchitecture ||
        getDefaultCpuArchitecture()) as CpuArchitecture,
      hasStaticIpNetworking: !!infraEnv?.staticNetworkConfig,
    }),
    [cluster?.cpuArchitecture, infraEnv?.cpuArchitecture, infraEnv?.staticNetworkConfig],
  );

```

Reply:
Ok, I've changed it